enhancements/monitoring/hypershift-monitoring.md

```diff
@@ -0,0 +1,384 @@
+
+> NOTE: We will focus on monitoring specific components in this document, but you will see logging parts on some diagrams. This is because logging can follow exactly the same pattern. Still, we will focus on monitoring (metrics) for now as it’s a bit unique (e.g. alerting part).
+
+#### The Big Picture
+
+For HSRE persona, we propose to send all relevant metrics for HSRE to the "Centralized Observability Product” using well-defined and stable APIs. At this point we call it product because by design this is replaceable component that can be mixed and matched according to the needs.
+
+For example:
+
+* For our Managed HSRE, this can be our RHOBS service.
+* For HSREs other than SD members, it can be locally deployed [Observatorium](http://observatorium.io/) or 3rd-party vendor like Grafana Cloud, Amazon or Google Managed Prometheus, Logz.io and others.
+
+A high-level view would look like this:
+
+![h](assets/hypershift-highlvl.png)
+
+The model as above allows:
+
+* Metrics Centralization
+* Extensibility (e.g. one can choose different Observability Products or switch them on demand).
+* Separation of concerns which should result in:
+  * Higher velocity of the Observability Product
+  * Higher velocity of OSD as the offering has to care less about in-cluster observability flows.
+  * Allows managing two products by separate teams without conflicts if one chooses to.
+
+> Note that this does not imply "Centralized Observability Product" to run in a separate cluster and be managed by a separate team. We can mix and match as needed, but strictly for Managed HyperShift use cases we would go for RHOBS service fully managed by Observability Group with AppSRE (SD).
+
+Reasons for choosing RHOBS for our HSRE:
+
+* We can have tighter control on observability budget and spending
+* It’s our common, Red Hat responsibility to offer managed OpenShift. If Observability Group + AppSRE will not provide their support and time, the same (or more) effort and money has to be spent on OSD side (or vendor)
+* The monitoring Team already manages this service.
+  * We already work closely together with AppSRE, tightly integrated with them and familiar with AppInterface.
+  * We have devs on-call and SRE mindset.
+
+## Design Details
+
+In details the full design looks as follows:
+
+![h](assets/hypershift-details.png)
+
+Let’s take a look at all parts:
+
+1. Hosted control planes does not deploy any own monitoring, other than relevant Service Monitors to be scraped.
+2. For all control planes we propose to deploy [Monitoring Stack Operator](https://github.com/rhobs/monitoring-stack-operator) that will deploy set of Prometheus/Prometheus Agents that will be sending data off cluster to RHOBS. No local alerting and local querying will be allowed. This path will forward both monitoring data as well as part of telemetry relevant for Telemeter.
```

Comment:
What is the requirement for prometheus here? How does the agent not suffice?

Reply:
👍🏽